src/main/java/core/basesyntax/dao/FruitDao.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.model.Fruit;
+
+public interface FruitDao {
+    void add(Fruit fruit);
```

Comment:
```suggestion
    Fruit add(Fruit fruit);
```

Reply:
Sorry but I don't get it. Why does this method should return the fruit instance? All that it does is adding a new fruit to the List of fruits. It changes the collection inside the method and I believe that it should not return the same fruit that we passed into it. Actually it's better to not return anything, isn't it? Please explain me what's the point of returning the fruit instance? For now I'm keeping the old method declaration but I will be glad to fix it if I'm wrong.